### ISSUE TITLE
Draft new information architecture

### ISSUE THREAD

**Issue by emileighoutlaw** [18F/fec-cms#306]
@jeanninehunter @edmullen and I are working on this.

Using what we've identified in the FEC content inventory (https://docs.google.com/spreadsheets/d/1LenmU9GkuHgQJPbvf60MIpxxLZ0GvU_TSJkkkF_sAWU/edit#gid=1605377718), we are drafting new potential content structures for betaFEC.

To do:
- [x] Meet week of May 15 to brainstorm and discuss next steps
- [x] Individual drafting of options
- [x] Meet May 23 to review draft proposals and decide on next steps

**Comment by emileighoutlaw** [comment]
This is an overdue readout of our Monday morning discussion of info architecture at FEC with @edmullen and @jeanninehunter 

(h/t Ed for articulating much of this very elegantly; I'm stealing many words from you)

### Background
Our analytics aren't awesome from FEC.gov, and even if it were awesome, content is spread out really weirdly, so we aren't sure how well individual pieces meet user needs.

### Idea
It comes down to this— we're going to have to make some (data-driven) assumptions about what content structures will make the most sense to users. We think the best way to get the richest amount of information about those assumptions is to shift to a model where we build out betaFEC with links to both our new beta content as well as legacy FEC.gov pages. Doing this will let us get feedback on navigational structures (and adjust our approach) before all the content is migrated to betaFEC (that's a long way away).
  
### Next steps
If we want to accomplish the idea above we need to:
- [ ] Create content groupings (based on what we know exists in the site from the content inventory) and pathways.
- [ ] Test those groupings and pathways with users. Refine. 
- [ ] Launch pathways on betaFEC. Test some more. Also track analytics. Iterate.

As we do all of the above, continue to rewrite and get updated content approved and moved to betaFEC.

@noahmanger @jenniferthibault— everything above impacts all of us. What are your thoughts on this approach and idea? Happy to answer questions and talk this through.

(Early 🚑  **unrefined**  🚑 Mural.ly content groupings Emileigh drafted: https://app.mural.ly/t/gsa6/m/gsa6/1463675441040)
(Ed's awesome IA thoughts document: https://docs.google.com/document/d/1a-Q-RnI58sGAM3viU0eTM4XTVTMgVDjbsPir5qQ-0ss/edit#)

**Comment by onezerojeremy** [comment]
Fundamentally, I really like the idea of focusing content effort on high priority pages, and doing something less involved with less used content. I will say that there's a bit of an user-expectation hit in the process of sending folks to the legacy site, but it might be worth it if there are things that we can't learn or do without doing that. 

For structure type questions, we can do some IA [tree-jack](https://www.optimalworkshop.com/treejack) testing to find out if the structure is making sense to folks without necessarily building it out. 

For other things, I guess the questions are what do we need to learn to move forward, and do we lose more experience points by directly porting over to the cms some of the less-consequential content that needs a solid (and porting other less used pages over (perhaps marking them in some way as such)) edit or sending folks to back to the legacy site?

**Comment by jenniferthibault** [comment]
@onezerojeremy The thing that seems complicated about "directly porting over to the cms some of the less-consequential content that needs a solid edit" is how do we port it over if we don't know where it lives in the new site yet? This would require simultaneous efforts of editing that content and redesigning the navigation.

I'm kind of suspecting that's part of why they've suggested linking back to the old site until we _do_ know if it's important enough to find a home for and how people expect to find it. Is that close, @emileighoutlaw ?

I'm not bothered by dings in experience of linking back to the old site: I've seen that a lot on university sites that get makeovers, and they don't even call themselves beta! Since we've done a good job of intro-ing the site as _beta_, I don't think it will make anyone rage quite in surprise frustration. But I don't know, we're all just hypothesizing. One way to find out!

(+fwiw, groupings and pathways seem like smart words for this, and words are hard)

**Comment by jenniferthibault** [comment]
Edit: not trying to brush off concerns about experience dings. Rereading it, it comes off that way. I should have said that I'm willing to accept the ding in experience for this low-priority, low-traffic content, in order to understand grouping & placement better.

**Comment by onezerojeremy** [comment]
totally. good points! 

I guess what I'm not understanding is why we need to tie the task of understanding the structure to the task of linking to content. We can test our assumptions about the structure and groupings without building. 

Apologies if I'm out of line here – possibly the coffee isn't working this week.

**Comment by edmullen** [comment]
@onezerojeremy I agree that we can learn about structure to an extent without linking to old content. We're currently putting together prototypes for just this reason. I think it is more about looking at navigation/findability/structure and content redesign as separate concerns that can be addressed in different timeframes.

Yes, this would be a significant philosophical shift. I see it as a choice between:

- **APPROACH 1: Beta should be a cordoned off experience in which all published aspects have been redesigned.** 
   - Once the entirety of beta is able to replace the current site, it is switched over. 
   - Beta experience is pure. 
   - Redesigned systems receive partial, opt-in traffic.
   - Delivers value to users of the beta site.

- **APPROACH 2: FEC.gov should operate with parts at different stages in the _legacy > beta > live/redesigned_ process.**
   - Once components of the site are proven to be more useful than the corresponding components, that component replaces the legacy version.
   - Experience is varied, and could be jarring, as users move between legacy and redesigned areas.
   - Redesigned components receive full audience traffic.
   - Delivers value to any users of redesigned components.
 
I prefer the second approach for a number of reasons.

- Improving findability is significant value we can deliver, and it's not dependent of improving the content.
- We prove value and benefit for more people sooner. 
   - Example: I believe the data side is currently more useful than the legacy site. Deferring sharing that usefulness with all data users (because they haven't chosen to opt into beta) until all other aspects are ready seems unfortunate.
- Beta currently is used by opt-in, early-adopter users. Once we sunset existing aspects of the current site and replace with our new experiences, they'll have to withstand more mainstream usage. It'll be "used in anger", as an English friend says, and we'll learn from people not as excited about change.
- I think there is a realistic chance that all aspects of the old site will not be redesigned or sunset for a while.

**Comment by onezerojeremy** [comment]
thanks Ed, that's helpful. I think your preference sounds totally reasonable and I appreciate your extra effort in explaining the ins and outs. 

My only remaining note is that I don't think we need to have designed prototypes to test structure, and in fact may be preferable

**Comment by noahmanger** [comment]
Super interesting stuff here, y'all. I know that there's strong interest in Approach 2 from our partners, and I agree with @edmullen 's assessment. Definite risks and challenges, but I think the upside is real. This approach also, by virtue of driving more traffic, will give us better analytics for those content decisions. 

I think we could start linking things back and forth immediately, though I think we're talking after the election before we start replacing things entirely (barring a change of plans). 

Also +1 to @onezerojeremy 's point about testing things with different means, though I know that analytics on real content will give us a fundamentally different understanding.

Last thought: we should spend some design time on managing the experience of going between the legacy and the new experiences. I agree that it probably isn't a _huge_ deal, but I think there's things we could do to make things better for users. Ideas / questions:
- Do we have a unique link style / icon associated with legacy links?
- Do we add a persistent header to the legacy site that links back to the beta if you came from the beta?
- Do we re-label `beta.fec.gov` to `new.fec.gov`? 
- How do we message when content on the legacy site has been replaced by content on the new site?

All in all, super interested in this approach and I'm sure we can figure out how to make it work for all involved. Let's find a way to test things tomorrow!

(Separately logging thoughts and questions re: Ed's doc in the doc).

**Comment by porta-antiporta** [comment]
Coming a bit late to the 🎉 where the 😎 folks are.

From a product/roadmap/vision perspective, Approach 2 forces the issue of "burning the ships" in that it virtually ensures that beta.fec loses its "beta" status and becomes live.   So I'm a fan.
One of the biggest risks to beta is the constant "it doesn't have all the content/functionality of fec.gov."  Approach 2 makes that a moot point because beta.fec will now not have that gap.

Also, +1 to @noahmanger's point around better analytics; as beta will be the entryway, we will have more control over analytics to identify what content is often sought vs. others.  This helps build a prioritized backlog that is more data driven than what we've been doing today.

What I'm curious about is how do we determine the initial categories/pathways and whether there's a balance that needs to be struck between gradually changing those (to abate user shock due to changes) and iterating those categories.  Note, I'm making the assumption that a "category" would be one of the links on the landing page (see @edmullen's IA doc)

**Comment by emileighoutlaw** [comment]
I'm responding via email because I'm having a very frustrating GitHub day
(I keep trying to post replies and GitHub times out 😤)

*Why I like door #2 and a future challenge*
I'm in favor of Approach 2 (I opened this issue, so that probably doesn't
shock y'all).

Beyond helping us get better data about possible IA structures, the
analytics will be important in helping content managers prioritize what to
tackle as we migrate to the new CMS and rewrite/combine old pages.

One potential sticky wicket of this approach is that the content is spread
so thinly it may be hard to link up the *concepts *we want to surface as
single pages on beta to the many pages that contain that information on
FEC.gov (This is the crappy part of content migration and why no content
migration is *actually moving pages from one place to another*).

That said, I do think this approach will work for many — probably even a
majority — of the pages on the current website. It's just a future
challenge, and we'll have several options to address it when we get there.

*How we determine the initial categories/pathways*
I think the brainstorming and testing we're doing this week on content
architecture is how we start tackling this!

*Noah's questions*
- Do we add a persistent header to the legacy site that links back to the
beta if you came from the beta?* I think so! A "take me back" thing. EITI
has one, and I think a couple other projects do too, if we want to poke
around for some inspiration.*
- Do we re-label beta.fec.gov to new.fec.gov? *Maaayyybbeee? *
- How do we message when content on the legacy site has been replaced by
content on the new site? *This might be a good place to apply our "archive"
header that we're going to apply to pages that live only as an archive and
not on the new site. Something that conveys "This is a snapshot of an old
page, provided as a historical record. We're maintaining a newer version on
betaFEC."*
